packages/amplication-client/src/Project/ProjectPage.tsx

```diff
@@ -4,28 +4,23 @@ import "./ProjectPage.scss";
 import ProjectSideBar from "./ProjectSideBar";
 import { AppRouteProps } from "../routes/routesUtil";
 import { match } from "react-router-dom";
+import PageContent from "../Layout/PageContent";
 
 type Props = AppRouteProps & {
   match: match<{
     workspace: string;
     project: string;
   }>;
 };
+const pageTitle = "Project";
```

Comment:
this can be passed by its router as another key in routeDef. then we can use it in all pages. also, I would like to take it out off pageContent (the helmet page title). it should be at App.tsx level